cypress/integration/main.spec.js

```diff
@@ -22,37 +51,74 @@ describe("Test insert functionality", () => {
 describe("Use Page Tests", () => {
     it("Visits the Use index, exercises controls", () => {
         cy.visit("/use/");
+        // Just to make sure the Cypress waits for Vue to render the page.
         cy.get("#masthead-logo").should("be.visible");
+        // Switching tabs.
         cy.findByRole("tab", { name: /Public Servers/i }).click();
         cy.get(".tab-pane.active").within(() => {
             cy.get("#public-server-filter-input").should("be.visible");
+            // Searching the table.
             cy.get("#public-server-filter-input").type("Galaxy Test");
             cy.findByText(/Beta version of Galaxy Main/i).should("be.visible");
+            // The pageStart/pageEnd/displayed calculation.
             cy.findByText(/Showing 1 to 1 of 1 results/i).should("be.visible");
             cy.get("#public-server-filter-input").clear();
             cy.findByText(/Showing .+ total/i).should("be.visible");
         });
     });
 
-    /* 
     // TODO: This is the test that will fail until the /use/ page is fixed.
     // This will pass against a develop server currently.
-
-    it("Tries to visit another page after /use/, via the masthead", () => {
+    it.skip("Tries to visit another page after /use/, via the masthead", () => {
```

Comment:
👍 Better way to skip than big comment blocks!  I knew this in the back of my mind from jest land and didn't think to try it in cypress.

Reply:
Yeah, very satisfying to discover that and use it instead of a big comment.